plugins/SitesManager/SitesManager.php

```diff
@@ -7,10 +7,7 @@
  *
  */
 namespace Piwik\Plugins\SitesManager;
-
-use Piwik\Common;
```

Comment:
I guess this missing use statement is causing UI build to fail... http://builds-artifacts.piwik.org/ui-tests.master/10881.7/screenshot-diffs/singlediff.html?processed=../processed-ui-screenshots/UIIntegrationTest_actions_pages.png&expected=UIIntegrationTest_actions_pages.png&github=UIIntegrationTest_actions_pages.png